<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ActiveExposureWidget</class>
 <widget class="QWidget" name="ActiveExposureWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>391</width>
    <height>110</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Exposure Active</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
  background-color: qlineargradient(x1:0, y1:0, x1:1, y1:6, stop: 0.9#39393b, stop:1#39393b);
  border:none;
  color:white;
}

QScrollBar:vertical {
  background: transparent;
  width: 13px;
  margin: 0px 0 0px 0;
}

QScrollBar::handle:vertical {
  background: gray;
  min-height: 13px;
  border-radius:6px;
}

QScrollBar::add-line:vertical {
  border: none;
  background: none;
  height: 0;
  subcontrol-position: top;
  subcontrol-origin: margin;
  }

QScrollBar::sub-line:vertical {
  border: none;
  background: none;
  height: 0;
  subcontrol-position: top;
  subcontrol-origin: margin;
  }

QScrollBar::up-arrow:vertical, QScrollBar::down-arrow:vertical {
  border: none;
  width: 0;
  height: 0;
  background: none;
}

QScrollBar::add-page:vertical, QScrollBar::sub-page:vertical {
  background: none;
}

QScrollBar:horizontal {
  background: transparent;
  height: 13px;
  margin: 0px 0 0px 0;
}

QScrollBar::handle:horizontal {
  background: gray;
  min-width: 13px;
  border-radius:6px;
  }

QScrollBar::add-line:horizontal {
  border: none;
  background: none;
  height: 0;
  subcontrol-position: left;
  subcontrol-origin: margin;
  }

QScrollBar::sub-line:horizontal {
  border: none;
  background: none;
  width: 0;
  subcontrol-position: left;
  subcontrol-origin: margin;
  }

QScrollBar::up-arrow:horizontal, QScrollBar::down-arrow:horizontal {
  border: none;
  width: 0;
  height: 0;
  background: none;
}

QScrollBar::add-page:horizontal, QScrollBar::sub-page:horizontal {
  background: none;
}

QPushButton{
  color:rgb(21,21,21);
  border: 1px solid rgb(94, 94, 94);
  background-color:rgb(125,129,133);
  min-height:30px;
  border-radius:6px;
}

QPushButton:hover{
  background-color:rgb(164, 168, 172);
}

QPushButton:pressed{
  background-color:rgb(166, 166, 166);
}

QLineEdit{
  border-radius:6px;
  border:1px solid rgb(89,89,89);
  background-color:rgb(37,38,39);
  color:white;
  padding: 1px 0px 1px 3px;
}

QLabel{
  background:none;
}

QListWidget{
  background-color:rgb(45,45,45);
  border:1px solid rgb(25,25,25);
  border-radius:4px;
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>254</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="1" rowspan="2">
    <widget class="QListWidget" name="m_ListWidget"/>
   </item>
   <item row="3" column="0">
    <widget class="QCheckBox" name="m_InvertCheckBox">
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QCheckBox{
max-width:100px;
border-radius:12px;
spacing:-71;
min-width:100px;
min-height:25px;
border:1px solid rgb(89,89,89);
}

QCheckBox:pressed{
background-color:rgb(149, 149, 149);
}

QCheckBox:hover{
background-color:rgb(139, 140, 142);
}

QCheckBox:checked {
background-color:rgb(149, 149, 149);
}

QCheckBox::indicator{
border:none;
width:100px;
}

QCheckBox::disabled{
 background-color: rgb(39, 39, 39);
 color: rgb(59, 59, 59);
 border: 1px solid rgb(59, 59, 59);
}</string>
     </property>
     <property name="text">
      <string>Invert</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Exposure Active</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QCheckBox" name="m_ActiveCheckBox">
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QCheckBox{
max-width:100px;
border-radius:12px;
spacing:-71;
min-width:100px;
min-height:25px;
border:1px solid rgb(89,89,89);
}

QCheckBox:pressed{
background-color:rgb(149, 149, 149);
}

QCheckBox:hover{
background-color:rgb(139, 140, 142);
}

QCheckBox:checked {
background-color:rgb(149, 149, 149);
}

QCheckBox::indicator{
border:none;
width:100px;
}

QCheckBox::disabled{
 background-color: rgb(39, 39, 39);
 color: rgb(59, 59, 59);
 border: 1px solid rgb(59, 59, 59);
}</string>
     </property>
     <property name="text">
      <string>Active</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
